beetmoverscript/task.py

```diff
@@ -4,11 +4,15 @@
 import re
 import shutil
 import scriptworker.client
+
+from copy import deepcopy
+
+from beetmoverscript import utils
```

Comment:
Am curious, do you have a preference of using `submodule.function` for the ease of reading or is there another reason?

Reply:
Actually, that's because of the monkeypatch used in the unittests. I wish we could use `from module.submobile import function`. If we define so, then momnkeypatch fails to patch. For more info: https://github.com/pytest-dev/pytest/issues/2229

Reply:
Ah, okay, didn't know this.  Thanks for pointing this out!